wallet/src/ui/app/redux/slices/sui-objects/index.ts

```diff
@@ -9,10 +9,13 @@ import {
 } from '@reduxjs/toolkit';
```

Comment:
FYI note that I have updated the line 26 and 27 in https://github.com/MystenLabs/sui/pull/2087/files#diff-465c7023c3f71c9fa7cd35a4429999d81c1322a59967e56b5b728209d2d8fa43 . There should not be any merge conflict. For local development, you need to rebase and rebuild your local rpc server binary(i.e., cargo build). I have tested locally that it works well with this PR

![CleanShot 2022-05-30 at 16 17 33](https://user-images.githubusercontent.com/76067158/171067672-2fb1b86f-b98f-49c8-af92-96dabd1105bc.png)
![CleanShot 2022-05-30 at 16 17 40](https://user-images.githubusercontent.com/76067158/171067690-9dd29d96-f1b3-43cd-b411-1734d06e03df.png)